src/datasource/OGC.js

```diff
@@ -1163,6 +1164,59 @@ class OGC extends ngeoDatasourceDataSource {
     return share;
   }
 
+  /**
+   * Compare active dimensions filters of two data sources. As Openlayers
+   * WFS format does not allow constructing multiple typenames GetFeature
+   * request with different filters we need to combine data sources depending
+   * on active dimensions filters.
+   * @param {!ngeox.dataSource.OGC} dataSource Remote data source to
+   *     compare with this one.
+   * @return {boolean} Whether the two data sources have the same active
+   *     dimensions filters. If both have no dimensions, they are considered
+   *     to be sharing the same dimensions filters.
+   * @export
+   * @override
+   */
+  haveTheSameActiveDimensionsFilters(dataSource) {
+    const myConfig = this.dimensionsFiltersConfig;
+    const itsConfig = dataSource.dimensionsFiltersConfig;
+    if (myConfig === null || itsConfig === null) {
+      return false;
+    }
+
+    const equals = function(key) {
```

Comment:
```suggestion
    const equals = (key) => {
```

and this `bind(this)` can be removed bellow